README.md

```diff
@@ -0,0 +1,54 @@
+# WeBloc
+
+> WeBloc is an open-source, community-driven event finder built on the MEAN Stack. Never has it been easier to find local events within your community.
+
+## Team
+
+  - __Product Owner__: Mark Keith
+  - __Scrum Master__: Trevor Pace
+  - __Development Team Members__: Nassir Zeinu
+
+## Table of Contents
+
+1. [Usage](#Usage)
+1. [Requirements](#requirements)
+1. [Development](#development)
+    1. [Installing Dependencies](#installing-dependencies)
+    1. [Tasks](#tasks)
+1. [Team](#team)
+1. [Contributing](#contributing)
+

```

Comment:
Even the hyperlinks that are clickable don't do anything, really. We should fix these.